src/platform/android/DnssdImpl.cpp

```diff
@@ -306,6 +325,8 @@ void InitializeWithObjects(jobject resolverObject, jobject browserObject, jobjec
 
     sBrowseMethod = env->GetMethodID(browserClass, "browse", "(Ljava/lang/String;JJLchip/platform/ChipMdnsCallback;)V");
 
+    sStopBrowseMethod = env->GetMethodID(browserClass, "stopDiscover", "(J)V");
```

Comment:
you also need check whether sStopBrowseMethod == nullptr? right? similar to sResolveMethod

Reply:
Right